sandbox/OpenIddict.Sandbox.AspNetCore.Client/Startup.cs

```diff
@@ -148,6 +149,14 @@ public void ConfigureServices(IServiceCollection services)
                            options.SetClientId("bXgwc0U3N3A3YWNuaWVsdlRmRWE6MTpjaQ")
                                   .SetClientSecret("VcohOgBp-6yQCurngo4GAyKeZh0D6SUCCSjJgEo1uRzJarjIUS")
                                   .SetRedirectUri("https://localhost:44381/callback/login/twitter");
+                       })
+                       .UseCognito(options =>
+                       {
+                           options.SetClientId("COGNITO_CLIENT_ID")
```

Comment:
Note: if you don't plan to add "real" credentials, please remove the changes made to the sample (only providers with credentials that can be used as-is are expected to be added in the sandbox)